charmcraft.yaml

```diff
@@ -1,15 +1,49 @@
 # Copyright 2024 Canonical Ltd.
 # See LICENSE file for licensing details.
-# This file configures Charmcraft.
-# See https://juju.is/docs/sdk/charmcraft-config for guidance.
+
+name: jenkins-agent
+title: Jenkins agent
+summary: Jenkins agent machine charm
+description: |
+  A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+  deploying and managing [Jenkins](https://www.jenkins.io/) Agent on machines and
+  configurable to use a Jenkins charm deployed in another Juju model.
+
+  This charm simplifies initial deployment and "day N" operations of Jenkins Agent
+  on VMs and bare metal.
+
+  As such, the charm makes it easy for those looking to take control of their own
+  Agents whilst keeping operations simple, and gives them the freedom to deploy on
+  the platform of their choice.
+
+  For DevOps or SRE teams this charm will make operating Jenkins Agent simple and
+  straightforward through Juju's clean interface. It will allow easy deployment
+  into multiple environments for testing changes, and supports scaling out for
+  enterprise deployments.
+links:
+  contact: https://launchpad.net/~canonical-is-devops
+  documentation: https://discourse.charmhub.io/t/jenkins-agent-documentation-overview/12694
+  issues: https://github.com/canonical/jenkins-agent-operator/issues
+  source: https://github.com/canonical/jenkins-agent-operator
+  website: https://charmhub.io/jenkins-agent
 
 type: charm
-bases:
-  - build-on:
-      - name: ubuntu
-        channel: "22.04"
-    run-on:
-      - name: ubuntu
-        channel: "20.04"
-      - name: ubuntu
-        channel: "22.04"
+
+base: ubuntu@24.04
+
+platforms:
+  amd64:
+    build-on: amd64
+    build-for: amd64
+  arm64:
```

Comment:
arm64?

Reply:
Yes this is a machine charm so it should be able to support arm64 as long as or CI allows it.